app/boards/shields/splitkb_aurora_sweep/Kconfig.defconfig

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2022 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+if SHIELD_SPLITKB_AURORA_SWEEP_LEFT
+
+config ZMK_KEYBOARD_NAME
+	default "Aurora Sweep"
+
+config ZMK_SPLIT_ROLE_CENTRAL
+	default y
+
+endif # SHIELD_SPLITKB_AURORA_SWEEP_LEFT
+
+if SHIELD_SPLITKB_AURORA_SWEEP_LEFT || SHIELD_SPLITKB_AURORA_SWEEP_RIGHT
+
+config ZMK_SPLIT
+	default y
+
+config ZMK_RGB_UNDERGLOW
+	select WS2812_STRIP
```

Comment:
I like this, we should consider refactoring other boards/shields to do this

Reply:
Yes! I did need to revert the same change for the display stuff, due to nice!view wanting it *not* to select ssd1306 directly.